packages/components/src/navigator/router/link.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * WordPress dependencies
+ */
+import { createElement, forwardRef } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import RouterContext from './router-context';
+import {
+	normalizeToLocation,
+	resolveToLocation,
+} from './utils/location-utils.js';
+
+// React 15 compat
+function isModifiedEvent( event ) {
+	return !! (
+		event.metaKey ||
+		event.altKey ||
+		event.ctrlKey ||
+		event.shiftKey
+	);
+}
```

Comment:
Do we need to support React 15?

Reply:
I don't think so. I removed some support as I was porting from g2, but this remained. I think we can remove it safely.